.changeset/salty-laws-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@bigcommerce/catalyst-core": minor
+---
+
+Refactor Category PLP in preparation for dynamic IO, by making sure there are no blocking calls other than static requests.
```

Comment:
Can we think/provide some sort of guide to this changelog on how to migrate to this new functionality?